frontend/src/scenes/notebooks/NotebookPanel/notebookPanelLogic.ts

```diff
@@ -26,10 +27,15 @@ export const notebookPanelLogic = kea<notebookPanelLogicType>([
             ['openSidePanel', 'closeSidePanel'],
             notebookPopoverLogic,
             ['setPopoverVisibility'],
+            notebooksModel,
+            ['deleteNotebook'],
         ],
     }),
     actions({
-        selectNotebook: (id: string, autofocus: EditorFocusPosition | undefined = undefined) => ({ id, autofocus }),
+        selectNotebook: (id: string, options: { autofocus?: EditorFocusPosition; silent?: boolean } = {}) => ({
```

Comment:
`silent` as a prop name confused me for a little bit. Wonder if there's something else we could call it or add a description for what it achieves. Maybe `open` or more explicitly `openInSidePanel`